_assets/build/Dockerfile-prune

```diff
@@ -0,0 +1,15 @@
+# Build status-go in a Go builder container
+FROM golang:1.10-alpine as builder
+
+RUN apk add --no-cache make gcc musl-dev linux-headers
+
+RUN mkdir -p /go/src/github.com/status-im/status-go
+ADD . /go/src/github.com/status-im/status-go
+RUN cd /go/src/github.com/status-im/status-go && make statusd-prune
+
+# Copy the binary to the second image
+FROM alpine:latest
+
+RUN apk add --no-cache ca-certificates bash
+
+COPY --from=builder /go/src/github.com/status-im/status-go/build/bin/statusd-prune /usr/local/bin/
```

Comment:
Do we want to specify default command: `CMD ["statusd-prune"]`?

Reply:
@adambabik I added `ENTRYPOINT` so that you can pass command line flags:

`docker run --rm -ti statusteam/statusd-prune -help`

Reply:
That's also fine. 

I like keeping `ENTRYPOINT` default and use default `CMD` because then I can do `docker run -it --rm statusteam/statusd-prune /bin/bash`. With this solution, `--entrypoint` needs to be changed to run a different program.

Reply:
yeah it's true. but then if I specify `CMD ["statusd-prune"]`, I can't run the container just passing `-dbpath=xyz`, but I still need to specify command and flags. 
Unless we want to hardcode the flags as well. Or is there another way to do that?

Reply:
@pilu one way to achieve this is to use:

`CMD statusd-prune $OPTS` 

It's used often in the Java word like so:

`CMD java $JAVA_OPTS -jar blah.jar`

Does not work with array version.

(not arguing for it, just an option)

Reply:
right, so then running it with this?

`docker run --rm -e OPTS="-db foo.db -upper 1" statusteam/statusd-prune`

Reply:
correct, you pass config through the environment

Reply:
But what's wrong with `docker run -it --rm statusteam/statusd-prune statusd-prune -dbpath="xyz"` :) It's a bit more characters but explicit on the other hand. `ENTRYPOINT` is fine as well, we can totally keep it.

Reply:
@adambabik yeah for me it's fine running the container with `image-name command args`, but then maybe we don't even need a `CMD`. Did you propose it just to have a kind of documentation to understand what's the main "executable" to call?

Reply:
I updated it to use `statusd-prune` as default `CMD`. And we can use it with:

`docker run -it --rm statusteam/statusd-prune statusd-prune -dbpath="xyz" -upper=N`